src/common/options/global.yaml.in

```diff
@@ -7675,11 +7732,17 @@ options:
   type: bool
   level: dev
   default: true
+  fmt_desc: Enables direct i/o to the journal. Requires ``journal block
+   align`` set to ``true``.
   with_legacy: true
 - name: journal_aio
   type: bool
   level: dev
   default: true
+  versionchanged: 0.61 Cuttlefish
```

Comment:
you could drop the versionchanged support, this is the only place we've used it and it's not important to note a change in cuttlefish anymore.

It seems release notes and different branches in the docs are better places for noting configuration changes

Reply:
dropped.